tools/raiddit/generate_claims.py

```diff
@@ -45,7 +45,15 @@ def hub(
 
     if not isinstance(output_file, Path):
         output_file = Path(output_file)  # type: ignore
-    create_hub_jsonl(signer, token_network_address, chain_id, hub_address, addresses, output_file)
+
+    create_hub_jsonl(
```

Comment:
Could he useful to also expose the direct channel feature on the cli.

But I guess it's not important right now.

Reply:
Yeah, let's skip it for now. I just need that code for the SP right now.